extensions/smallrye-graphql/deployment/src/main/java/io/quarkus/smallrye/graphql/deployment/SmallRyeGraphQLConfig.java

```diff
@@ -13,29 +14,17 @@
     String rootPath;
 
     /**
-     * The path where GraphQL UI is available.
-     * The value `/` is not allowed as it blocks the application from serving anything else.
-     */
-    @ConfigItem(defaultValue = "/graphql-ui")
-    String rootPathUi;
-
-    /**
-     * Always include the UI. By default this will only be included in dev and test.
-     * Setting this to true will also include the UI in Prod
-     */
-    @ConfigItem(defaultValue = "false")
-    boolean alwaysIncludeUi;
-
-    /**
-     * If GraphQL UI should be enabled. By default, GraphQL UI is enabled.
+     * Enable metrics
      */
-    @ConfigItem(defaultValue = "true")
-    boolean enableUi;
+    @ConfigItem(name = "metrics.enabled", defaultValue = "false")
+    boolean metricsEnabled;
 
     /**
-     * Enable metrics
+     * Config group for all UI related options.
+     * Configuration properties for UI
```

Comment:
This javadoc will be the title of the section so maybe just "User interface" or "UI configuration" would be better.